Installation/Schema/xxllnc-zaaktype-post.json

```diff
@@ -0,0 +1,65 @@
+{
+    "$id": "https://development.zaaksysteem.nl/schema/xxllnc.zaakTypePost.schema.json",
+    "$schema": "https://docs.commongateway.nl/schemas/Entity.schema.json",
+    "version": "0.5",
```

Comment:
version should be 0.0.1 ?
![image](https://github.com/user-attachments/assets/61760c7a-46c5-49a6-851d-ebe65c3d2e91)
please delete this file